kyo-core/shared/src/test/scala/kyoTest/typeMapTest.scala

```diff
@@ -0,0 +1,116 @@
+package kyoTest
+
+import kyo.*
+
+class typeMapTest extends KyoTest:
```

Comment:
Can you add tests to show that it's not possible to call `get` and `add` with intersection and union type parameters?

Reply:
Done.